### ISSUE TITLE
Ubuntu 20: UI 

### ISSUE THREAD

**Issue by AivanF** [tomaspinho/rtl8821ce#159]
I installed Ubuntu Desktop 20 on a fresh Lenovo ThinkCentre M715q, but there was no WiFi option settings. I installed driver just as described in this repo. After that I disabled Secure Boot (in BIOS), and WiFi got working fine. A few days later I launched the computer without available WiFi networks, and after logging in, the UI completely disappeared. I guess, the driver found no WiFi network and unexpectedly died (possibly this is specific to the Ubuntu 20). I restarted multiple times, enabled Secure Boot, but result is still the same: the screen dies on successful logging in :( Is it possible to solve the problem without reinstalling the OS and save my files?

**Comment by tomaspinho** [comment]
@AivanF this driver should not cause UI issues like the one you're describing. I suggest booting a live environment and backing up your files before reinstalling Ubuntu.

**Comment by AivanF** [comment]
@tomaspinho Yes, this is really strange. But I have already reinstalled the OS, even multiple times and different versions of Ubuntu, the result is the same: after installing the driver, logging in stops working. I'm not experienced in system programming, and have no idea what is the origin of the problem, but there is no doubt about correlation with the driver. Maybe the installation process breaks graphical drivers? Or some other indirect impact...

**Comment by wunzo** [comment]
I experienced this problem too after installing the driver. Reinstalling OS without installing driver fixed it.

**Comment by n3z0xx** [comment]
I experienced same problem on Pop_OS 21.04 (kernel 5.13.0-7260). But sometimes it's even drops system to kernel panic - attempted to kill init. Also sometimes after starting from suspend system fezzes for exactly 5 mins. If you tell me how, I can provide any logs and help as much i can.
